packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -516,7 +555,7 @@ class _SelectableRegionState extends State<SelectableRegion> with TextSelectionD
   late Offset _selectionStartHandleDragPosition;
   late Offset _selectionEndHandleDragPosition;
 
-  late List<TextSelectionPoint> points;
+  late List<TextSelectionPoint> _points;
```

Comment:
This is probably a mistake I made previously, This can be function variable instead.

Reply:
Oh right, I'll do that.